azure/dt-azure-log-forwarder-terraform-deployment/main.tf

```diff
@@ -0,0 +1,32 @@
+provider "azurerm" {
+  features {}
```

Comment:
It's recommended to specify the provider version. For instance:
```hcl
terraform {
  required_providers {
    azurerm = {
      source  = "hashicorp/azurerm"
      version = "3.72.0"
    }
  }
}
```

Reply:
I think provider version is there - azure/dt-azure-log-forwarder-terraform-deployment/versions.tf